docker/install.sh

```diff
@@ -16,11 +16,12 @@
 
 cd ~/
 # Source Environment variables
-source /opt/ros/foxy/setup.bash
 source /home/carma/catkin/setup.bash
+source /opt/ros/foxy/setup.bash
 # Install dependencies
 sudo apt-get update
 rosdep update
 rosdep install --from-paths src --ignore-src -r -y
 # Build 
-colcon build --packages-up-to novatel_oem7_driver carma_novatel_driver_wrapper
\ No newline at end of file
+colcon build --packages-up-to novatel_oem7_driver carma_novatel_driver_wrapper
```

Comment:
no need for the --packages-up-to
do
```
colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release
```
instead

Reply:
Updated